web_export_view/static/src/xml/web_export_view_template.xml

```diff
@@ -2,11 +2,10 @@
 
 <templates id="template" xml:space="preserve">
 <t t-name="AddExportViewMain">
-    <div class="oe_form_dropdown_section">
-        <button class="oe_dropdown_toggle oe_dropdown_arrow">Export Current View</button>
-        <ul class="oe_dropdown_menu">
-            <li class="oe_sidebar_export_view_xls"><span>Excel</span></li>
-        </ul>
+    <div class="o_cp_sidebar btn-group btn-group-sm">
+        <button class="btn btn-default export_treeview_xls" type="button">
```

Comment:
Add a `title` to this to give a hint on what it does, or add some text next to the icon please.